src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -331,7 +331,15 @@ private function parseDefinition($id, $service, $file, array $defaults)
             }
         }
 
-        $tags = isset($service['tags']) ? $service['tags'] : array();
```

Comment:
> it adds 1 useless check

yes, but we don't care, readibility is more important here (thinking about the complexity of nested if/else block) IMHO